src/router/storeSync/storeSync.routerPlugin.js

```diff
@@ -64,6 +64,11 @@ function storeMutationWatcher(store, mutation, router) {
                     log.info('Error while routing to', query, error)
                     routeChangeIsTriggeredByThisModule = false
                 })
+            // if the short linked version of the URL is already defined,
+            // we must refresh it as the URL has just changed
+            if (store.state.position.shortLink) {
```

Comment:
Won't this regenerate the short-link with each map action (pan, zoom, etc) even if no short-link is visible?

Reply:
We need to be careful not to generate too many useless shortlink !

Reply:
yep, I think we will need to monitor in the state when the menu share section is opened (so when an update is needed if the URL changes)